frontend/src/metabase/core/components/EditableText/EditableText.styled.tsx

```diff
@@ -27,12 +28,16 @@ export const EditableTextRoot = styled.div<EditableTextRootProps>`
       border-color: ${color("border")};
     `}
 
-  &:after {
-    content: attr(data-value);
-    visibility: hidden;
-    white-space: pre-wrap;
-    word-wrap: break-word;
-  }
+  ${({ isEditingMarkdown }) =>
+    isEditingMarkdown &&
+    css`
+      &:after {
+        content: attr(data-value);
```

Comment:
couldn't we just set `content` dynamically based on the isEditingMarkdown property?